docs/en/writing-running-appium/tutorial/swipe/ios-mobile-screen.md

```diff
@@ -0,0 +1,92 @@
+## iOS 'mobile:': Screen swipe
+
+XCTest framework supports unique gestures like "mobile:scroll" and
+"mobile:swipe". They are not so flexible as UIScrollable but still
```

Comment:
It is good to leave a couple of links to pages like https://developer.apple.com/documentation/xctest/xcuielement/1618667-swipeup and https://developer.apple.com/documentation/xctest/xcuielement/1500758-scrollbydeltax here